osu.Game/Input/OsuUserInputManager.cs

```diff
@@ -10,7 +10,7 @@ namespace osu.Game.Input
 {
     public partial class OsuUserInputManager : UserInputManager
     {
-        protected override bool AllowRightClickFromLongTouch => PlayingState.Value == LocalUserPlayingState.NotPlaying;
+        protected override bool AllowRightClickFromLongTouch => PlayingState.Value != LocalUserPlayingState.Playing;
```

Comment:
Isn't the rest of the game interface technically available at the pause/fail screen? If you have a keyboard you can access settings, chat, user overlay, beatmap overlay, etc?

Reply:
Also, as far as I can tell this is a revert to `master`. That said, I haven't gone through the other cases like `GameplayScreenRotationLocker` which change logic from `master` (again, as far as I can tell).

We discussed yesterday what would it take to make it so that PR reviews didn't take many times longer than actually making changes - not making subtle changes like this would help.

Reply:
Fine to go with as you have it, will revisit in a future.